data/data-pipeline/data_pipeline/etl/constants.py

```diff
@@ -1,8 +1,13 @@
 DATASET_LIST = [
```

Comment:
Reordering this so that the longest-running ETLs start first. This will speed up parallelization.